src/bluesearch/database/download.py

```diff
@@ -141,6 +167,55 @@ def get_pubmed_urls(
     return urls
 
 
+def get_s3_urls(
+    bucket: boto3.resources.base.ServiceResource,
```

Comment:
Not sure actually. Our CI did not complain. Do you know why exactly PyCharm complains?

Reply:
It says `Cannot find reference 'resources' in '__init__.py' ` I guess it's because neither of the submodules of `boto3` were ever imported explicitly.